src/frontend/org/voltdb/importclient/Log4jSocketHandlerImporter.java

```
@@ -193,6 +202,13 @@ public void run()
                     if (!Log4jSocketHandlerImporter.this.callProcedure(new SaveLog4jEventInvocation(hostname, event, m_tableName))) {
                         Log4jSocketHandlerImporter.this.error("Failed to insert log4j event");
                     }
+                    if (m_hasBackPressure) {
+                        try {
+                            Thread.sleep(100);
+                        } catch (InterruptedException ioe) {
+                            //
```

should we really swallow this exception may we should rethrow it as sensible extension to `RuntimeException`